src/components/utils/Hooks.js

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from 'react';
```

Comment:
That's not a component don't use a capital letter in the filename, rename file to `hooks`